Master RMarkdown Document & Render Code/Locality_Profiles_Master_Markdown.Rmd

```diff
@@ -658,27 +658,30 @@ x <- x + 1
 
 **Summary**
 
-This section includes acute hospital data, delayed discharge bed days and A&E attendances. For the most recent time period available, `r LOCALITY` had:
+**For the most recent time periods available, `r LOCALITY` had:**
 
- - **`r latest_emergency_adm_loc`** emergency hospital admissions per 100,000 population, compared to `r scot_emergency_adm` in Scotland. 
+ - **`r latest_emergency_adm_loc1`** emergency hospital admissions per 100,000 population, compared to `r scot_emergency_adm1` in Scotland. 
  
- - **`r latest_bed_days_loc`** unscheduled acute specialty bed days per 100,000 population, compared to `r scot_bed_days` in Scotland.  
+ - **`r latest_bed_days_loc1`** unscheduled acute specialty bed days per 100,000 population, compared to `r scot_bed_days1` in Scotland.  
  
- - **`r latest_ae_att_loc`** A&E attendances per 100,000 population, compared to `r scot_ae_att` in Scotland. 
+ - **`r latest_ae_att_loc1`** A&E attendances per 100,000 population, compared to `r scot_ae_att1` in Scotland. 
  
- - **`r latest_dd_loc`** delayed discharge bed days per 100,000 population aged over 65, compared to `r scot_dd` in Scotland. 
+ - **`r latest_dd_loc1`** delayed discharge bed days per 100,000 population aged over 65, compared to `r scot_dd1` in Scotland. 
  
- - **`r latest_falls_loc`** emergency hospital admissions from falls per 100,000 population aged over 65, compared to `r scot_falls` in Scotland. 
+ - **`r latest_falls_loc1`** emergency hospital admissions from falls per 100,000 population aged over 65, compared to `r scot_falls1` in Scotland. 
  
- - **`r latest_read_loc`** emergency readmissions (28 days) per 1,000 discharges, compared to `r scot_read` in Scotland. 
+ - **`r latest_read_loc1`** emergency readmissions (28 day) per 1,000 discharges, compared to `r scot_read` in Scotland. 
  
- - **`r latest_ppa_loc`** potentially preventable hospital admissions per 100,000 population, compared to `r scot_ppa` in Scotland. 
-
+ - **`r latest_ppa_loc$formatted_data[2]`** potentially preventable hospital admissions per 100,000 population, compared to `r scot_ppa$formatted_data[2]` in Scotland. 
 
 ##### Page break
 
 ### Emergency Admissions
 
+Figure `r x` presents the emergency admissions rate per 100,000 population in the `r LOCALITY` locality from `r min_year_ea` to `r max_year_ea`. 
+
+As presented in Figure `r x`, the emergency admissions rate in the `r LOCALITY` locality for `r max_year_ea` is `r latest_emergency_adm_loc1`, a `r percent_rate_change`% `r word_change_rate` since `r min_year_ea`. The `r HSCP` HSCP rate is `r hscp_emergency_adm1`, a `r hscp_rate_change`% `r word_change_hscp` since `r min_year_ea`. The `r HB` health board rate is `r hscp_emergency_adm1` in `r max_year_ea`, a `r hb_rate_change`% `r word_change_hb` since `r min_year_ea` and the Scotland rate is `r scot_emergency_adm1`, a `r scot_rate_change`% `r word_change_scot` since `r min_year_ea`.
```

Comment:
This is quite a petty suggestion 😆 but if the rate change value starts with 8 or 18 the article should be 'an' not 'a'.

For example "a 17.1% increase ..." is fine but "a 18.1% ..." or "a 82.5% ..." should be "an 18.1% ..." or "an 82.5% ...".

This is probably an issue throughout the profiles, but we can tackle it here and worry about the rest 
later!

I suggest adding a new function (probably in General, for ease later).

```r
get_article <- function(number) {
  if (substr(number, 1, 1) == "8" || substr(number, 1, 2) == "18") {
    return("an")
  } else {
    return("a")
  }
}
```

Then update the text in the Markdown to:
```suggestion
As presented in Figure `r x`, the emergency admissions rate in the `r LOCALITY` locality for `r max_year_ea` is `r latest_emergency_adm_loc1`, `r get_article(percent_rate_change)` `r percent_rate_change`% `r word_change_rate` since `r min_year_ea`. The `r HSCP` HSCP rate is `r hscp_emergency_adm1`, `r get_article(hscp_rate_change)` `r hscp_rate_change`% `r word_change_hscp` since `r min_year_ea`. The `r HB` health board rate is `r hscp_emergency_adm1` in `r max_year_ea`, `r get_article(hb_rate_change)` `r hb_rate_change`% `r word_change_hb` since `r min_year_ea` and the Scotland rate is `r scot_emergency_adm1`, `r get_article(scot_rate_change)` `r scot_rate_change`% `r word_change_scot` since `r min_year_ea`.
```

Even writing this comment was a chunk of work, so possibly we just move this to an issue to be tackled as a separate task, so that it doesn't hold up the wider Community Care chapter improvements.

Reply:
This now a PR to this branch (#111) as well as a PR to apply it to the wider project (#110)